### DISCUSSION TITLE
Intel drivers

### DISCUSSION THREAD

**Discussion by AdrianKrajnik2** [Vanilla-OS/live-iso#383] [Q&A] [answered]
Hi, I want to ask how to install Intel drivers? I couldn't find a way to do this in the documentation. I found only how to install nvidia drivers.

**Comment by kbdharun** [comment, marked as answer]
Hi, Intel drives are installed by default in all our images and installer ISOs by default, as Intel's open-source firmware and drivers for almost all of their products are directly shipped with the latest linux kernels that we use so no separate user intervention is required from your side (and this is pretty much the case for all Linux distros).

There are specific things like their WiFi cards that are proprietary, that we have separately added to our [core image](https://github.com/Vanilla-OS/core-image/blob/dev/modules/05-firmware.yml) with the [`firmware-iwlwifi`](https://wiki.debian.org/iwlwifi) package and also things like microcode is shipped too, so you are GTG.

---

_A bit of history..._

Intel has a pretty nice track record with Linux support where they add the drivers and firmware for their upcoming hardware to the Linux Kernel before it's released so that when it gets released it will be supported by various Linux distributions and vendors to use with the hardware. AMD's catching up too, and has been open-sourcing their entire hardware stack in recent years.
